packages/react/src/components/builder-page.component.tsx

```diff
@@ -781,7 +781,7 @@ export class BuilderPage extends React.Component<
     const dataString =
       this.props.data && size(this.props.data) && hash(this.props.data)
     let key = Builder.isEditing ? this.name : this.props.entry
-    if (!Builder.isEditing && dataString && dataString.length < 300) {
+    if (key && !Builder.isEditing && dataString && dataString.length < 300) {
```

Comment:
`undefined:${hashOfObject}` this was causing two components of with the same url but different models to show the same content